### ISSUE TITLE
File URIs are not parsed

### ISSUE THREAD

**Issue by osa1** [mrkkrp/modern-uri#13]
I wasn't sure if `file:///...` is actually a valid URI so checked the RFC and it gives this example: `file:///etc/hosts` (section 1.1). However modern-uri can't parse this:

```
Prelude Text.URI> mkURI "file:///etc/hosts"
*** Exception: ParseException "file:///etc/hosts" (TrivialError (SourcePos {sourceName = "", sourceLine = Pos 1, sourceColumn = Pos 8} :| []) (Just (Tokens ('/' :| ""))) (fromList [Tokens ('%' :| ""),Tokens ('[' :| ""),Label ('A' :| "SCII alpha-numeric character"),Label ('i' :| "nteger"),Label ('u' :| "sername")]))
```

For comparison, `network-uri` parses this:

```
Prelude Network.URI> parseURI "file:///etc/hosts"
Just file:///etc/hosts
```

**Comment by mrkkrp** [comment]
Looks like a bug to me, then. I'll fix it next week and release a new version. Thanks for reporting this!

**Comment by mrkkrp** [comment]
Fixed in 0.2.2.0, now on Hackage.
